secure-a-cluster.md

```diff
@@ -20,45 +20,41 @@ Also, feel free to watch this process in action before going through the steps y
 ## Step 1.  Create security certificates
 
 ~~~ shell
-# Create a certs directory:
+# Create a certs directory and safe directory for the CA key.
+# If using the default certificate directory (`${HOME}/.cockroach-certs`), make sure it is empty.
 $ mkdir certs
+$ mkdir my-safe-directory
 
 # Create the CA key pair:
 $ cockroach cert create-ca \
---ca-cert=certs/ca.cert \
---ca-key=certs/ca.key
+--certs-dir=certs \
+--ca-key=my-safe-directory/ca.key
 
 # Create a client key pair for the root user:
 $ cockroach cert create-client \
 root \
---ca-cert=certs/ca.cert \
---ca-key=certs/ca.key \
---cert=certs/root.cert \
---key=certs/root.key
+--certs-dir=certs \
+--ca-key=my-safe-directory/ca.key
 
 # Create a key pair for the nodes:
 $ cockroach cert create-node \
 localhost \
 $(hostname) \
---ca-cert=certs/ca.cert \
---ca-key=certs/ca.key \
---cert=certs/node.cert \
---key=certs/node.key
+--certs-dir=certs \
+--ca-key=my-safe-directory/ca.key
 ~~~
 
 - The first command makes a new directory for the certificates.
-- The second command creates the Certificate Authority (CA) certificate and key: `ca.cert` and `ca.key`.
-- The third command creates the client certificate and key, in this case for the `root` user: `root.cert` and `root.key`. These files will be used to secure communication between the built-in SQL shell and the cluster (see step 5).
-- The fourth command creates the node certificate and key: `node.cert` and `node.key`. These files will be used to secure communication between nodes. Typically, you would generate these separately for each node since each node has unique addresses; in this case, however, since all nodes will be running locally, you need to generate only one node certificate and key.
+- The second command creates the Certificate Authority (CA) certificate and key: `ca.crt` and `ca.key`.
```

Comment:
Hmm, I see the alternate file extension here, too. Do we want to use `.crt` or `.cert`?